scripts/deploy_beeswax.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+set -e
+set -x
+
+SERVER_URL=$1
+DEPLOY_KEY=$2
+
+export GOPATH=/opt/beeswax_gopath
+INSTALL_PATH=$GOPATH/src/github.com/iankronquist/
+HONEYPOT_NAME=beeswax
+
+
+install_dependencies() {
+	apt-get update
+	apt-get -y install docker.io tcpdump gcc golang git supervisor python-pip
+	pip install docker-compose
+}
+
+install_project() {
+	if [ ! -e $INSTALL_PATH ] ; then
+		mkdir -p $INSTALL_PATH
+		git clone http://github.com/iankronquist/senior-project-experiment.git $INSTALL_PATH/$HONEYPOT_NAME
+	fi
+	cd $INSTALL_PATH/$HONEYPOT_NAME
+	make all
+}
+
+# The docker group must already exist
+# The project must already be cloned
+make_user() {
+
+        #Adds docker group if it doesn't exist
+      	getent group docker || groupadd docker
+
+
+	id -u 'beeswax'|| useradd -d /home/beeswax -s /bin/bash -m beeswax -g users -G docker
+	chown -R beeswax $INSTALL_PATH
+}
+
+make_configs() {
+        echo "!!!!!!!!THIS IS IT BOYS!!!!!!!!"
+
+        wget $SERVER_URL/static/registration.txt -O registration.sh
+        chmod 755 registration.sh
+        # Note: this will export the HPF_* variables
+        ./registration.sh $SERVER_URL $DEPLOY_KEY "beeswax"
```

Comment:
Add the dot back, I thought it was a typo, didn't know that it meant source, Ubuntu 14.04 wasn't working correctly